core/frontend/src/components/version-chooser/VersionChooser.vue

```diff
@@ -596,6 +602,28 @@ export default Vue.extend({
       }
       return this.available_versions.local.some((image) => image.sha === sha)
     },
+    get_input_file(): File | null {
+      const { files } = document.getElementById('file') as HTMLInputElement
+
+      if (files == null || files.length <= 0) {
+        return null
+      }
+
+      return files[0]
+    },
+    is_file_input_not_empty(v: File | null): true | string {
+      this.file_input_error = ''
+      return !!v || this.input_file_required_message
+    },
+    validateInputFileForm(): boolean {
```

Comment:
The style should follow the others functions as well

Reply:
Changed both the methods and the computed property to follow the convention, only error messages remained snake case due the others data be in this way also